src/audio/smart_amp/smart_amp.c

```diff
@@ -84,6 +84,7 @@ static inline int smart_amp_alloc_memory(struct smart_amp_data *sad,
 		goto err;
 
 	hspk = sad->mod_handle;
```

Comment:
Care to explain why?

Reply:
bzero is actually equivalent to 'memset_s'. https://github.com/thesofproject/sof/blob/master/src/arch/xtensa/include/arch/string.h#L24
I saw several use cases of bzero in the project and just thought it is better choice. Thanks.

Reply:
Looks like bzero is deprecated, (and we could deprecate it here too). To quote the man page
```
CONFORMING TO
       The  bzero()  function  is deprecated (marked as LEGACY in POSIX.1-2001); use memset(3) in new programs.  POSIX.1-2008 removes
       the specification of bzero().  The bzero() function first appeared in 4.3BSD.
```
So safe to remove this patch from the PR.

Reply:
OK. I will remove bzero from the PR.
Fyi, I will keep the last patch to add memset for the two pointers, not zero initialized.